postcss.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = ({ env }) => {
+  const shouldOptimize = env === "production" ? {} : false;
+
+  return {
+    parser: false,
+    plugins: {
+      autoprefixer: shouldOptimize,
+      "postcss-merge-rules": shouldOptimize,
+      "postcss-discard-duplicates": shouldOptimize,
+      "postcss-merge-longhand": shouldOptimize,
+      "postcss-discard-comments": shouldOptimize
```

Comment:
I believe we can pass the configuration to minimize to webpack instead.
In the css-loader something like
```
options: {
         minimize: true
    }
```

Reply:
As far as I get [the docs](https://github.com/webpack-contrib/css-loader#minimize) you can pass in cssnano configuration there, but these things are postcss (which we need for autoprefixer). Feel free to do a follow-up PR to also add cssnano, would love to see the effects on the output